### PR TITLE
feat: Retrieve data from pokemon api and they were shown in main view

### PR DESCRIPTION
**Pull request by JonathanAmador12**
- Create VM
- Create PokemonModel, this model how information will look like.
- Create CreatePokemonRequest, this structure specify all the infomation need to retrieve pokemon page.

### CODE REVIEW

File: PokedexV2/VIewModel/PokemonViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//  PokedexV2
+//
+//  Created by JonathanA on 12/10/22.
+//
+
+import Foundation
+
+
+class PokemonViewModel: ObservableObject {
+    @Published var pokemons: [PokemonModel] = []
```
Comment (AmadorHeE) [inline PokedexV2/VIewModel/PokemonViewModel.swift]: It should be pokemon

File: PokedexV2/VIewModel/PokemonViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    
+    func getPokemos() {
+        let httpRequestHandler = HTTPRequestHandler()
+        let pokemonPageRequest = PokemonPageRequest()
+        httpRequestHandler.requestData(requestData: pokemonPageRequest) {[weak self] result in
+            switch (result){
+            case .success(let pokemos):
```
Comment (AmadorHeE) [inline PokedexV2/VIewModel/PokemonViewModel.swift]: The same, it should be Pokemon

File: PokedexV2/Views/PokemonView.swift [hunk 3]
```diff
@@ -6,34 +6,39 @@
...
-    
     var body: some View {
         ZStack {
-            Color(.white)
+            LinearGradient(gradient: Gradient(colors: [Color("background1"), Color("background2"), Color("background3")]), startPoint: .top, endPoint: .bottom)
             VStack {
                 ScrollView {
                     LazyVGrid(columns: gridForm, spacing: 20) {
-                        ForEach(0...100, id: \.self) { numero in
+                        ForEach(vm.pokemons, id: \.self) { pokemon in
```
Comment (AmadorHeE) [inline PokedexV2/Views/PokemonView.swift]: id: \.self is not necessary anymore

**Review by AmadorHeE — CHANGES_REQUESTED** [review]

File: PokedexV2/VIewModel/PokemonViewModel.swift (hunk 1 above)
Comment (JonathanAmador12) [reply PokedexV2/VIewModel/PokemonViewModel.swift]: it's done

**Review by AmadorHeE — APPROVED** [review]

**Pull request merged**